From aaa2fdc879c4332a3062ac6a64496251f8091ae9 Mon Sep 17 00:00:00 2001
From: Zygmunt Krynicki <me@zygoon.pl>
Date: Tue, 22 Aug 2017 17:39:40 +0200
Subject: [PATCH 1/4] cmd/snap-seccomp: link to libseccomp dynamically

On many distributions we canont link to libseccomp dynamically so before
this is addressed more comprehensively upstream, link libseccomp
dynamically via a distro patch.

Signed-off-by: Zygmunt Krynicki <me@zygoon.pl>
---
 cmd/snap-seccomp/main.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/snap-seccomp/main.go b/cmd/snap-seccomp/main.go
index ed0ef7f4d..170f9bbdd 100644
--- a/cmd/snap-seccomp/main.go
+++ b/cmd/snap-seccomp/main.go
@@ -21,7 +21,7 @@ package main
 
 //#cgo CFLAGS: -D_FILE_OFFSET_BITS=64
 //#cgo pkg-config: --static --cflags libseccomp
-//#cgo LDFLAGS: -Wl,-Bstatic -lseccomp -Wl,-Bdynamic
+//#cgo LDFLAGS: -lseccomp
 //
 //#include <asm/ioctls.h>
 //#include <ctype.h>
-- 
2.14.1

